projects/fsastorefrontlib/src/cms-components/my-account/address-book/address-form/address-form.component.html

```diff
@@ -156,13 +154,20 @@
       </div>
     </div>
   </div>
-  <div class="cx-checkout-btns row">
-    <div class="col-md-12 col-lg-6" *ngIf="showCancelBtn">
-      <button class="btn btn-block btn-action" (click)="back()">
+  <div class="cx-checkout-btns row col-md-12 col-lg-12">
```

Comment:
Seems  col classes are not needed here. As a rule div with row class shouldn't have col classes at the same time.